app-services/secrets/README.md

```diff
@@ -48,49 +128,23 @@ Success! Disabled the secrets engine (if it existed) at: secret/
 Success! Enabled the kv secrets engine at: secret/
 ```
 
-#### Insecure mode
-
-When security is disabled, the secrets can be written to and are retrieved from the *Writable.InsecureSecrets* section of the configuration file. Insecure secrets and their paths can be configured as below.
-
-```toml
- [Writable.InsecureSecrets]    
-      [Writable.InsecureSecrets.NoPath]
-        Path = ''
-        [Writable.InsecureSecrets.NoPath.Secrets]
-          username = 'nopath-user'
-          password = 'nopath-pw'
-      
-      [Writable.InsecureSecrets.AWS]
-        Path = 'aws'
-        [Writable.InsecureSecrets.AWS.Secrets]
-          username = 'aws-user'
-          password = 'aws-pw'
-      
-      [Writable.InsecureSecrets.MongoDB]
-        Path = 'mongodb'
-        [Writable.InsecureSecrets.MongoDB.Secrets]
-          username = ''
-          password = ''
-```
-
-`NOTE: An empty path is a valid configuration for a secret's location  `
-
-#### Run StoreSecrets and GetSecrets
+5. Run the secrets service: `go run main.go`
+6. Follow the instructions in [Storing and Getting Secrets](#storing-and-getting-secrets) in order to test storing and retrieving secrets from the secret store.
 
-##### StoreSecrets
+## Storing and Getting Secrets
 
-When running in secure mode, secrets for can be stored in the secret store (Vault) by making an HTTP `POST` call to `http://[host]:[port]/api/v1/secrets`.  
+These tests use a collection of Postman requests, in *SecretsExample.postman_collection.json*, to store and retrieve secrets from the secret store.
 
-*SecretsExample.postman_collection.json* contains Postman requests to store secrets to using the */secrets* API route. 
+1. Import the collection *SecretsExample.postman_collection.json* into Postman.
 
-If in secure mode, execute the Store Secrets Postman requests to push secrets to Vault.
+2. Execute the `Store Secrets` request in the Postman collection to push secrets to Vault.
```

Comment:
Add that this is going thru the App Service REST API, not directly to Vault so that the secret is exclusively for that app service instance.